test/unit_tests/folding/__snapshots__/folding.test.ts.snap

```diff
@@ -387,10 +355,6 @@ exports[`Testing folding ranges of test_record.vhd 1`] = `
 
 exports[`Testing folding ranges of test_stateMachine.vhd 1`] = `
 [
-  {
-    "endLine": 5,
-    "startLine": 4,
-  },
   {
```

Comment:
Why is this gone?

Reply:
Because the folding now works more generalized on the following:

- declarations
- statements
- useclauses (incl. library)
- interface lists
- special blocks like instantiations, association lists, case (generate), types (record and protected)

Reply:
Okay. But why can you no longer fold design entities?

Reply:
Because then regions often overlap and vscode will throw away some of the regions. For example entities with statements will behave awkwardly when whole entities can be collapsed. Imo this is cleaner

Reply:
Okay, I added folding for configurations and for entities, architectures and packages (bodies) the folding is sufficient.